src/Juvix/Compiler/Internal/Language.hs

```diff
@@ -230,8 +230,9 @@ data Pattern
 
 instance Hashable Pattern
 
-newtype InductiveParameter = InductiveParameter
-  { _inductiveParamName :: VarName
+data InductiveParameter = InductiveParameter
+  { _inductiveParamName :: VarName,
+    _inductiveParamImplicit :: IsImplicit
```

Comment:
Why do we need this? Isn't it the case that all inductive parameters must be explicit?

Reply:
I think I've been overzealous here. There is some indication if a parameter is implicit/explicit in Internal or earlier, and I just propagated it down to Core. Ultimately, it's not really used. Do you think I should revert the propagation of implicity information for parameters?

Reply:
I think you are referring to the `Abstract` language, where we `Abstract.FunctionParameter` (which has `_paramImplicit`) instead of defining a separate `InductiveParameter` data type without the `isImplicit` field. 
Since we are not going to have implicit parameters for inductive types in the near future, I'd say it is better to revert the change. And since we plan on removing `Abstract`, the anomaly there will also be taken care of

Reply:
Okay, I'll do that then

Reply:
done